test/mechanics/constraintsPde/ConstraintCompanionPde.cpp

```diff
@@ -0,0 +1,195 @@
+#include "BoostUnitTest.h"
+#include <fakeit.hpp>
+//#include <cmath>
+//#include "mechanics/groups/Group.h"
+#include "mechanics/constraintsPde/ConstraintCompanion.h"
+
+using namespace fakeit;
+using namespace NuTo;
+using namespace NuTo::ConstraintPde;
+
+
+//// Helper functions for comparisons
+namespace NuTo
+{
+namespace ConstraintPde
+{
+bool operator==(const Term& a, const Term& b)
+{
+    if (&(a.GetNode()) != &(b.GetNode()))
+        return false;
+    if (a.GetCoefficient() != b.GetCoefficient())
+        return false;
+    if (a.GetComponent() != b.GetComponent())
+        return false;
+    return true;
+}
+
+bool operator==(const Equation& a, const Equation& b)
+{
+    if (a.GetRhs(0.0) != b.GetRhs(0.0))
+        return false;
+    if (a.GetRhs(42.0) != b.GetRhs(42.0))
+        return false;
+    if (a.GetTerms() != b.GetTerms())
+        return false;
+    return true;
+}
+
+} // Constraint
+} // NuTo
+
+
+// quite the stupid tests, but oh well...
```

Comment:
??

Reply:
--> git blame in the original testfile 
Just copied, didn't dare to much more than to get it running again ;p